### PR TITLE
Contextual message branch

### PR DESCRIPTION
**Pull request by gsuarez17**
### What does it do?

Describe the technical changes you did.

### Why is it needed?

Describe the issue you are solving.

### How to test it?

Below are listed every action that should be possible within the application.

#### Content Manager

**Article**

List view:

- [ ] Publication Status is working
- [ ] Review Workflows assignees & filtering is working (EE)

Entry view:

- [ ] CKEditor field is working
- [ ] Assignees and Stages are working (EE)

**Page**

Entry view:

- [ ] Creating and publishing a page is working

#### Plugins

**Content-type Builder**

- [ ] Content-type Builder is working

**Media Library**

- [ ] Media Library is working (upload)
- [ ] Folders are working

#### Settings

- [ ] Marketplace is working
- [ ] API Tokens Interface
- [ ] Internationalization Interface
- [ ] Single Sign-On Settings
- [ ] Review Workflows
  - Show custom created Workflow
  - Drag N Drop is working
  - Add new workflow stage is possible
  - Colors drop down is working
- [ ] Webhooks
  - Review Workflows Stage Change option
- [ ] Audit Logs
  - List view
  - Single log
- [ ] Roles
  - Default Roles
  - Create new role
  - Show CRUD + Publish
    - Field level
    - Locale
- [ ] Users & Permissions
  - Roles 
  - Add New Role

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

File: src/app/components/shared/contextual-message/contextual-message/contextual-message.component.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+$neutral-title: var(--hc-color-paysafe-blue-50);
+
+$positive-accent: var(--hc-color-positive-accent);
+$positive-title: var(--hc-color-positive-title);
+
+$warning-accent: var(--hc-color-warning-accent);
+$warning-title: var(--hc-color-warning-title);
+
+$danger-accent: var(--hc-color-danger-accent);
+$danger-title: var(--hc-color-danger-title);
```
Comment (acamachoaltimetrik) [inline src/app/components/shared/contextual-message/contextual-message/contextual-message.component.scss:12]: can we move this to the `variables.scss` to maintain all the scss variables just in one place?

File: src/app/components/shared/contextual-message/contextual-message/contextual-message.component.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import { Component,Input } from '@angular/core';
+import { MatCard, MatCardContent, MatCardHeader, MatCardTitle } from '@angular/material/card';
+import { of } from 'rxjs';
+
+export enum MessageType {
+  Neutral = 'neutral',
+  Positive = 'positive',
+  Warning = 'warning',
+  Danger = 'danger',
+}
```
Comment (acamachoaltimetrik) [inline src/app/components/shared/contextual-message/contextual-message/contextual-message.component.ts:11]: magnificent!

File: src/app/components/shared/contextual-message/contextual-message/contextual-message.component.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  templateUrl: './contextual-message.component.html',
+  styleUrl: './contextual-message.component.scss'
+})
+export class ContextualMessageComponent {
+  readonly DETAIL_STYLE_STRING: string = "-detail";
+  readonly TITLE_STYLE_STRING: string = "-title";
+  @Input() title: string = 'Message Title';
+  @Input() message: string = 'nadfbasdlkjfnalskjdfnlaksdjflnaksdfnasdkjfnalksdjfnalkjsdfnlaksdfnlkajsdfnkaljsdfnlakaksdfnasdkjfnaljfnalkjjfnalkjsdfnlaksdfnlkajsdfnkaljsdfnlakaksdfnasdkjfnalksdjfnalkjsdfnlaksdfnlkajsdfnkaljsdfnlak';
+
+  private _messageValue: MessageType = MessageType.Neutral;
```
Comment (acamachoaltimetrik) [inline src/app/components/shared/contextual-message/contextual-message/contextual-message.component.ts:26]: can we make this to be an input?

File: src/app/components/shared/contextual-message/contextual-message/contextual-message.component.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  standalone: true,
+  imports: [MatCard,NgClass],
+  templateUrl: './contextual-message.component.html',
+  styleUrl: './contextual-message.component.scss'
+})
+export class ContextualMessageComponent {
+  readonly DETAIL_STYLE_STRING: string = "-detail";
+  readonly TITLE_STYLE_STRING: string = "-title";
+  @Input() title: string = 'Message Title';
+  @Input() message: string = 'nadfbasdlkjfnalskjdfnlaksdjflnaksdfnasdkjfnalksdjfnalkjsdfnlaksdfnlkajsdfnkaljsdfnlakaksdfnasdkjfnaljfnalkjjfnalkjsdfnlaksdfnlkajsdfnkaljsdfnlakaksdfnasdkjfnalksdjfnalkjsdfnlaksdfnlkajsdfnkaljsdfnlak';
```
[the remark this answers is not in the document]
Comment (gsuarez17) [reply src/app/components/shared/contextual-message/contextual-message/contextual-message.component.ts:24]: Haha yes I meant to take that out entirely! my bad.

**Pull request closed without merging**